src/test/regress/expected/pg13.out

```diff
@@ -13,38 +13,42 @@ SET citus.next_shard_id TO 65000;
 CREATE TABLE dist_table (name char, age int);
 CREATE INDEX name_index on dist_table(name);
 SELECT create_distributed_table('dist_table', 'name');
- create_distributed_table
----------------------------------------------------------------------
-
+ create_distributed_table 
+--------------------------
+ 
 (1 row)
 
 SET client_min_messages to DEBUG1;
 SET citus.log_remote_commands to ON;
 -- make sure vacuum parallel doesn't error out
 VACUUM (PARALLEL 2) dist_table;
 NOTICE:  issuing VACUUM (PARALLEL 2) test_pg13.dist_table_65000
-DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+DETAIL:  on server postgres@localhost:57637 connectionId: 3
 NOTICE:  issuing VACUUM (PARALLEL 2) test_pg13.dist_table_65001
-DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+DETAIL:  on server postgres@localhost:57638 connectionId: 4
 VACUUM (PARALLEL 0) dist_table;
 NOTICE:  issuing VACUUM (PARALLEL 0) test_pg13.dist_table_65000
-DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+DETAIL:  on server postgres@localhost:57637 connectionId: 3
 NOTICE:  issuing VACUUM (PARALLEL 0) test_pg13.dist_table_65001
-DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+DETAIL:  on server postgres@localhost:57638 connectionId: 4
```

Comment:
not sure why we have these changes, it seems like normalize isn't run

Reply:
You need to run `ci/normalize_expected.sh` yourself

Reply:
yeah that makes sense, I think it was because `make reindent` already runs `ci/normalize_expected.sh` and I forgot to run it :)